projects/com.oracle.truffle.llvm.parser.api/src/com/oracle/truffle/llvm/parser/api/model/symbols/constants/AbstractConstant.java

```diff
@@ -39,6 +40,8 @@ protected AbstractConstant(Type type) {
         this.type = type;
     }
 
+    public abstract void accept(AbstractConstantVisitor visitor);
```

Comment:
_AbstractConstant_ right now only serves the purpose of storing the type so the individual constants do not have to implement that functionality themselves. The actual top-level interface for constants is _Constant_. I think we should either remove the interface altogether and rename this class to _Constant_ or declare the _accept()_ method in the interface.